Agile-Software-Engineering-Team-1/sportacus
# Feature/user auth frontend

This implements user creation from the frontend to the backend. POST request performed via axios. I removed the saved teams in the db for now since this was such a mess to get working, so the database will probable need to be pulled in. 
To test:
Start the backend server -
                                          python ./manage.py runserver
Start the frontend server - 
                                          npm start
Login to django auth with username: admin password: Password123! to keep track of users
On react side, navigate to the signup page using the header and input information.
You should see a POST on the django side, along with the new user in the database
